cv32/sim/uvmt_cv32/Makefile

```diff
@@ -61,24 +109,30 @@ DSIM_IMAGE              = dsim.out
 # UVM Environment
 export DV_UVMT_CV32_PATH = $(PROJ_ROOT_DIR)/$(COREV_PROJ)/tb/uvmt_cv32
 export DV_UVME_CV32_PATH = $(PROJ_ROOT_DIR)/$(COREV_PROJ)/env/uvme_cv32
-export TBSRC_HOME        = $(PROJ_ROOT_DIR)/$(COREV_PROJ)/tb
-#DV_UVMT_CV32_SRCS  = $(wildcard $(DV_UVMT_CV32_PATH)/*.sv))
+DV_UVMT_CV32_SRCS  = $(wildcard $(DV_UVMT_CV32_PATH)/*.sv))
```

Comment:
We don't want this here.  We inherited this technique from the pulp-platform Makefiles.  OpenHW manifests should be controlled in the *_pkg.sv and *.flist source files, not the Makefile.